 src/CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index cb1f8c7..19450b4 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,7 +1,7 @@
 include_directories("${PROJECT_SOURCE_DIR}/src")
 include_directories("${PROJECT_BINARY_DIR}")
 include(GenerateExportHeader)
-add_library(edit_static STATIC editline.c fn_complete.c history.c)
+add_library(edit_a STATIC editline.c fn_complete.c history.c)
 add_library(edit SHARED editline.c fn_complete.c history.c libedit.def)
 set_target_properties(edit PROPERTIES PREFIX "")
 GENERATE_EXPORT_HEADER(edit
@@ -15,13 +15,13 @@ target_link_libraries(edit_test edit)
 add_executable(edit_test_dll libedit_test_dll.c)
 install (TARGETS edit edit_test edit_test_dll
   DESTINATION "${CMAKE_SOURCE_DIR}/bin${LIB_SUFFIX}")
-install (TARGETS edit_static
+install (TARGETS edit_a
   DESTINATION "${CMAKE_SOURCE_DIR}/lib${LIB_SUFFIX}")
 install (FILES editline/readline.h
   DESTINATION "${CMAKE_SOURCE_DIR}/include/editline")
 string(TOUPPER "${CMAKE_BUILD_TYPE}" uppercase_CMAKE_BUILD_TYPE)
 if (MSVC AND uppercase_CMAKE_BUILD_TYPE MATCHES "DEBUG")
-  install (FILES ${CMAKE_CURRENT_BINARY_DIR}/Debug/edit_static.pdb
+  install (FILES ${CMAKE_CURRENT_BINARY_DIR}/Debug/edit_a.pdb
     DESTINATION "${CMAKE_SOURCE_DIR}/lib${LIB_SUFFIX}")
   install (FILES ${CMAKE_CURRENT_BINARY_DIR}/Debug/edit.pdb
     DESTINATION "${CMAKE_SOURCE_DIR}/bin${LIB_SUFFIX}")
